mate-scripts/src/commands/Command.ts

```diff
@@ -1,9 +1,14 @@
+import path from 'path';
 import { getConfig } from '../tools';
 import { Config, ProjectTypes } from '../typedefs';
 
 export abstract class Command {
   protected readonly rootDir: string;
 
+  protected readonly binDir: string;
+
+  protected readonly mateScriptsDir: string;
```

Comment:
unneeded. You should always use the bin. There are no files from the repo in the homework node_modules